src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/diff/PrisonerMovementsEventService.kt

```diff
@@ -28,7 +28,7 @@ class PrisonerMovementsEventService(
   private val domainEventEmitter: HmppsDomainEventEmitter,
   private val telemetryClient: TelemetryClient,
 ) {
-  fun generateAnyMovementEvents(
+  fun generateAnyEvents(
     previousPrisonerSnapshot: Prisoner?,
```

Comment:
minor name change to so all services simply have a `generateAnyEvents` method, the type is implied by the service name